src/components/Resources/index.js

```diff
@@ -5,23 +5,44 @@ import PersonalMenu from '../PersonalMenu';
 import Search from '../Search';
 import Grid from '@material-ui/core/Grid';
 import { ResourceCard } from './ResourceCard';
+import * as helper from '../helper';
 
 function Resources({ getResourcesUrl }) {
   const [resources, setResources] = useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState(null);
 
   useEffect(() => {
     axios
       .get(getResourcesUrl)
       .then(function(response) {
         // handle success
-        setResources(response.data.results);
+        setResources(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
   }, []);
 
+  const search = searchValue => {
+    setSearchValue(searchValue);
+    setLoading(true);
+    setErrorMessage(null);
+    axios
+      .get(helper.buildQueryString(getResourcesUrl, searchValue))
```

Comment:
I'm unsure if `getResourceUrl` is worth being in a separate function as of now. If anything, maybe should apply the same pattern that we have established with the Resource page and do something similar as the `getResource` query

https://github.com/codebuddies/frontend/blob/main/src/utils/queries.js

Reply:
Is there somewhere in the code base that stores constants, or simple helper methods that should be shared?  I wanted to start one via this example so that future code changes aren't so painful.  If there is a better way please let me know?

Reply:
Oh!  util didn't exist when we wrote this code.  I'm not tied to any particular location for globals, just that there is one somewhere.

Reply:
fwiw `getResourcesUrl` won't be a thing anymore once the following is merged in: https://github.com/codebuddies/frontend/pull/147/files

... but let's get this PR cleared up first, and I can rebase in that other PR!